src/commands/deployImage/imageSource/containerRegistry/acr/createAcr/RegistryNameStep.ts

```diff
@@ -29,14 +29,14 @@ export class RegistryNameStep extends AzureWizardPromptStep<CreateAcrContext> {
         if (name.length < minLength || name.length > maxLength) {
             return localize('validationLengthError', 'The name must be between {0} and {1} characters.', minLength, maxLength);
         } else if (!/^[a-z][a-zA-Z0-9]*$/.test(name)) {
-            return localize('validateInputError', `Connection names can only consist of alphanumeric characters.`);
+            return localize('validateInputError', `The name must consist of alphanumeric characters.`);
```

Comment:
If we're going to make the error more generic anyway, may as well just use your validation utils here.

Reply:
I agree, I just need to add the upper-case option because right now it only supports lower-case alphanumeric.  I figured I'd do that in a separate PR, maybe even after release.  I just wanted to change the message because I thought `Connection` might have been a typo